tests/resource_test.py

```diff
@@ -17,3 +18,22 @@ def test_static_files():
         "myapp-schema not in list={}",
         x,
     )
+
+
+def test_render_resource():
+    from sirepo import resource
+    from pykern import pkunit
+
+    d = pkunit.data_dir()
+    pkunit.file_eq(
+        d.join("expect.py"),
+        actual_path=resource.render_resource(
+            "actual.py",
+            d,
```

Comment:
This parameter is actually a bug. It shouldn't work this way. Resources should always come from the package_data directory. In this case, you'll need to construct something so that it is found with package_path.

To make this simpler, you could pick an arbitrary sirepo resource like test_static_files does. One way to do this is to create something like `sirepo/package_data/resource_test_data` that is small and then you don't have to setup package_path (which is unnecessarily complicated). This costs a little bit in complexity of package_data having test data, but it is a more realistic test. The file in resource_test_data could even be a README.txt.jinja that includes some documentation (and a single jinja expansion) that tells you why this particular file is there.

You don't necessarily need to have a `tests/resource_data` directory for this test. Just test some text that you know should be there along with the jinja value expansion, e.g. `pkre("var=abc")`. The file_eq tends to be a bit heavy for these types of problems, because there's too much copying and not enough context within the python part of the test.

Try this suggestion without thinking too hard about it. If it doesn't work, we can talk about it.